doc/reduce-traffic.md

```diff
@@ -46,7 +46,7 @@ Be reminded of the effects of this setting.
 
 - Fee estimation will no longer work.
 - It sets the flag "-walletbroadcast" to be "0", only if it is currently unset.
-  Doing so disables the automatic broadcasting of transactions from wallet. Not
+  Doing so disables the automatic broadcasting of transactions from a wallet. Not
```

Comment:
This is specifically from the built-in wallet (not, for example, from ZecWallet full node when used with a separate zcashd).

Reply:
That makes sense. You could also consider "the" wallet, which is used a few lines down.